### ISSUE TITLE
Fury Warrior: Recommending raging blow on 5+ targets with bonegrinder buff active

### ISSUE THREAD

**Issue by GarrettCottrell** [herotc/hero-rotation#834]
HeroRotation Version #9.2.0.06
HeroDBC Version #9.2.0.01
HeroLib Version 9.2.0.01

Talents: 3/3/1/3/3/3/1
Conduits: Piercing Verdict/Merciless Bonegrinder/Depths of insanity

The addon does not appear to support the merciless bonegrinder buff's changes to furry warrior AOE rotation on 5+ targets

The issue seems to be in the APL itself - it is not optimized for the merciless bonegrinder buff (even on raidbots) - it is most certainly wrong to raging blow on 8 targets during bonegrinder buff but the raidsbots apl is doing just that and I think that may be where this issue is stemming from?

A recommendation would be to not mark this profile as "good" and maybe mark it as WIP due to the apl not being optimized for aoe (unless you all think you can change the addon despite the apl not being optimized)

**Comment by Cilraaz** [comment]
The [Simulationcraft APL](https://github.com/simulationcraft/simc/blob/shadowlands/engine/class_modules/sc_warrior.cpp) currently has the following priorities for Whirlwind and Raging Blow.

1. Whirlwind to put up the Meat Cleaver buff on 2+ targets.
2. Raging Blow (regardless of target count) to keep up the Will of the Berserker buff or when using Tier28 2pc or at 2 charges.
3. Whirlwind when Merciless Bonegrinder is active with 3+ targets.
4. Whirlwind as a filler.

So, it appears they've taken Merciless Bonegrinder into account. I'm not knowledgeable enough about Fury to say if the APL is correct or not. If you feel the APL is in error, though, you may wish to submit an issue ticket on [their github](https://github.com/simulationcraft/simc/issues) or mention it in [their Discord channel](https://discord.com/channels/244067101478879234/244072792948080641).

**Comment by GarrettCottrell** [comment]
so on 5 targets with the merciless bonegrinder buff active, I am being recommened raging blow (I do not have the will of the berserker legendary equipped) by the addon. If the apl is taking tthat into acount then the recommendation I am recieving is incorrect - so the error may be in the implementation but not in the apl?

**Comment by Cilraaz** [comment]
No. Point 2 in my previous response. If you have 2 charges of Raging Blow or 2 pieces of tier 28 gear, it'll be suggested before Whirlwind with Merciless Bonegrinder.

**Comment by GarrettCottrell** [comment]
Ah okay, so yes, I just wanted to make sure that it was doing the behavior I thought. I did bring this up in the discord and it was agreed that is not optimal behavior during the bonegrinder buff. I was told that the maintainers of this apl do not choose to keep it fully optimal due to the difficulty with aoe and optimizing for executes. 

During the bonegrinder buff, it should be recommending whirlwind (even when raging blow is capped) at 5 or more targets. I have attached a screenshot as an example of very suboptimal behavior. The apl is essentially ignoring bonegrinder and proceeding with normal aoe (prio'ing raging blow when it is capping - this is correct when the bonegrinder buff is not active). Correct behavior with bonegrinder buff active would be to replace the raging blows, during the bonegrinder buff, with whirlwind and only interupt whirlwind spam to refresh enrage with rampage. 

![8targetsample](https://user-images.githubusercontent.com/49660931/158077539-00126811-171b-4680-b17f-b666eb71134a.PNG)
